src/pagination/pagination.ts

```diff
@@ -7,41 +7,46 @@ import {getValueInRange, toInteger, toBoolean} from '../util/util';
 @Component({
   selector: 'ngb-pagination',
   changeDetection: ChangeDetectionStrategy.OnPush,
+  styles: [`
+    button.page-link {
```

Comment:
I'm not sure this is the best selector and I'm a bit worried that this one is going to "leak" outside of this widget. Any other ideas?